packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.scss

```diff
@@ -4,103 +4,131 @@
   .SelectList {
     @include inputStyles;
     font-size: 14px;
-    line-height: 14px !important;
+    line-height: 20px;
     border: 1px solid $neutral-200 !important;
-    border-radius: 6px !important;
+    border-radius: $border-radius-corners-wider;
     min-height: 40px;
     max-height: 40px;
     padding: 10px 16px;
-  
-    .cwsl__control {
+
+    [class$='-control'] {
       gap: 8px;
       letter-spacing: 0.02em;
       width: 100%;
       flex-wrap: wrap;
       align-items: flex-start !important;
-      cursor: text !important;
-  
-      .cwsl__value-container {
+
+      [class$='-ValueContainer'] {
```

Comment:
in other component (CWTypeaheadSelectList) we use `.tasl__value-container` and other classes with `cwsl` prefix. I think it would be good to be consistent here, no? Or what is the reason to change it?

Reply:
Thanks for the review, @masvelio!
CWSelectList might be used as a base component for another component. In such a case, the new component will probably have a unique classname prefix, for styles specific to it. I still want the basic styling that goes with a select list to be available to that new component, hence the continued use of attribute selectors in `CWSelectList.scss` and the use of classname prefixes in the typeahead's styles. I do think, though, the presence of a default classname prefix on `CWSelectList` might be a bit confusing, given my reasoning. Hence, I'll remove it.